ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -321,14 +344,14 @@ main()
 	if [ -z "${8}" ]; then
 		MotionRegression="${G_DEFAULT_MOTION_REGRESSION}"
 	else
-		MotionRegression="${8}"
+		MotionRegression=$(interpret_as_bool "${8}")
 	fi
 
 	local DeleteIntermediates
 	if [ -z "${9}" ]; then
 		DeleteIntermediates="${G_DEFAULT_DELETE_INTERMEDIATES}"
 	else
-		DeleteIntermediates="${9}"
+		DeleteIntermediates=$(interpret_as_bool "${9}")
 	fi
 
 	# Turn MotionRegression into an appropriate numeric value for fix_3_clean
```

Comment:
Can't we delete this section, as long as we put the setting of `MotionRegression` and `DeleteIntermediates` through the `interpret_as_bool` function as well, in the default case?

Reply:
Stylistic choice, I prefer to only assign to any variable once, to make it conceptually easier to track, though this situation is borderline.  However, putting both cases through interpret_as_bool may be a good idea for anyone who edits a default.

Reply:
I'm not sure if my main point got across here.  Now that both cases go through `interpret_as_bool`, why should ReApplyFixPipeline.sh have a section of code after this (L357-380) in which it checks again whether `MotionRegression` and `DeleteIntermediates` are appropriately set to 0 or 1?  We don't have that bit of redundancy in ReApplyFixMultiRunPipeline.

Reply:
Indeed I had missed the lines below the one you attached the comment to, as I only looked at the comment in the conversation tab.  Should be resolved by e9eaaff7abcdeeb565596c76e2e6ccd7cacca818.